DRAFT.md

```diff
@@ -1051,28 +1072,19 @@ constexpr indirect& operator=(const indirect& other);
 
 1. _Mandates_: `T` is a complete type.
```

Comment:
Similarly to the copy-constructor, should this be also mandating that `is_copy_constructible_v<T>`?

Should it also be mandating that `is_copy_assignable_v<T>`?